README.md

```diff
@@ -40,10 +40,10 @@ Single-Page Application with React server-side rendering. [Demo store](https://d
 
 ## Installation
 
-- [with GitHub](https://github.com/cezerin2/cezerin2/blob/master/docs/using-source-code.md)
+- [with GitHub](https://github.com/cezerin2/cezerin2-mono/blob/master/docs/using-source-code.md)
 - [with Docker](https://github.com/cezerin2/cezerin2/blob/master/docs/using-docker.md)
-- [How to deploy a Cezerin2 on Ubuntu 16.04](https://github.com/cezerin2/cezerin2/blob/master/docs/how-to-deploy-a-cezerin2-on-ubuntu-16-04.md)
-- [How to deploy a Cezerin2 on Ubuntu 18.04.1 (from GitHub)](https://github.com/cezerin2/cezerin2/blob/master/docs/how-to-deploy-a-cezerin2-on-ubuntu-18-04-1-github.md)
+- [How to deploy a Cezerin2-mono on Ubuntu 16.04](https://github.com/cezerin2/cezerin2-mono/blob/master/docs/how-to-deploy-a-cezerin-on-ubuntu-16-04.md)
```

Comment:
-a-cezerin?  How about just -cezerin and it's, capital C on github/Cezerin2/cezerin2-mono

Reply:
Rest are good for me on this file.

Reply:
This "-a-cezerin" renamed to "cezerin2-mono", but this "github/Cezerin2/cezerin2-" is not possible to rename, it's from github set pages to repo.